apps/services/search-indexer/src/migrate/lib/elastic.ts

```diff
@@ -165,3 +169,99 @@ export const removeIndexIfExists = async (indexName: string) => {
 
   return false
 }
+
+/**
+ * Returns all document ids and popularity scores from a given index
+ *
+ * @param {string} oldIndex
+ */
+export const getAllPopularityScores = async (index: string) => {
+  const client = await esService.getClient()
+  const scores = []
+
+  // do a scroll query to get a snapshot of the index
+  let data = await client.search<SearchResponse<MappedData>, unknown>({
+    index,
+    body: {
+      query: {
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        match_all: {},
```

Comment:
We might be able to replace `match_all: {}` with something like:
```JSON
"range": {
  "popularityScore": {
    "gt": 0
  }
}
```
This would reduce payload and remove need for checks in code